resources/views/livewire/settings/price-lists.blade.php

```diff
@@ -0,0 +1,134 @@
+                     x-on:data-table-row-clicked="$wire.showEditModal($event.detail.id)"
+                >
+                    <livewire:data-tables.price-list-list />
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <x-modal.card wire:model.defer="editModal">
+        <x-slot name="title" class="text-lg leading-6 font-medium text-gray-900 dark:text-white">
+            {{ ($selectedPriceList['id'] ?? false) ? __('Edit Price List') : __('New Price List') }}
+        </x-slot>
+        <div class="space-y-8 divide-y divide-gray-200">
+            <div class="space-y-8 divide-y divide-gray-200">
+                <div>
+                    <div class="mt-6 grid grid-cols-1 gap-y-6 gap-x-4 sm:grid-cols-6">
+                        <div class="space-y-3 sm:col-span-6">
+                            <x-input wire:model="selectedPriceList.name" :label="__('Name')"/>
+                            <x-select wire:model="selectedPriceList.parent_id" :label="__('Parent')"
+                                :options="$priceLists" option-value="id" option-label="name" />
+                            <div x-show="priceList.parent_id > 0 " class="grid grid-cols-1 gap-y-6">
+                                <x-input wire:model="selectedPriceList.discount.discount" :label="__('Discount')"/>
+                                <x-toggle wire:model="selectedPriceList.discount.is_percentage" lg :label="__('Is Percentage')"/>
+                            </div>
+                            <x-input wire:model="selectedPriceList.price_list_code" :label="__('Code')"/>
+                            <x-toggle wire:model="selectedPriceList.is_net" lg :label="__('Is Net')"/>
+                            <x-toggle wire:model="selectedPriceList.is_default" lg :label="__('Is Default')"/>
+                        </div>
+                    </div>
+                </div>
+                <div>
+                    <x-table>
+                        <x-slot name="title">
+                            <h2 class="pt-4">{{ __('Product category discounts') }}</h2>
+                        </x-slot>
+                        <x-slot name="header">
+                            <th>{{ __('Category') }}</th>
+                            <th class="w-44">{{ __('Discount') }}</th>
+                            <th class="w-16">{{ __('%') }}</th>
+                            <th class="w-14"></th>
+                        </x-slot>
+                        <template x-for="(productCategory, index) in productCategories">
+                            <tr>
+                                <td class="text-center">
+                                    <div x-text="productCategory.name" class="mr-2"></div>
+                                </td>
+                                <td>
+                                    <div class="flex justify-center">
+                                        <x-input x-model="productCategory.discounts[0].discount"/>
```

Comment:
`<x-inputs.number x-model="productCategory.discounts[0].discount"/>`

please divide by 100 in livewire component if its percentage and multiply by 100 on load, for the user its not intuitive to add a below 0 value